src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DTCInterfaces/IPrepareInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace System.Transactions.DtcProxyShim.DTCInterfaces;
+
+/// <summary>
+/// The IPrepareInfo interface is superseded by the IPrepareInfo2 interface.
+/// </summary>
+/// <remarks>
+/// See https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms686533(v=vs.85).
+/// </remarks>
+[ComImport, Guid("80c7bfd0-87ee-11ce-8081-0080c758527e"), InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+internal interface IPrepareInfo
+{
+    /// <summary>
+    /// The GetPrepareInfoSize method is called by the resource manager to determine the size in bytes of the prepare information that needs to be logged.
+    /// </summary>
+    /// <param name="pcbPrepInfo">
+    /// A pointer to the size in bytes of the prepare information. The actual prepare information is available by a call to <see cref="GetPrepareInfo" />.
+    /// </param>
+    void GetPrepareInfoSize(out ulong pcbPrepInfo);
```

Comment:
This isn't correct. `long` and `ulong` in .NET are 64-bit. However, on Windows, the native `LONG` and `ULONG` are 32-bits. This should instead be `out uint`.

Reply:
Thanks, it's been a while :)